apps/website/package.json

```diff
@@ -24,10 +24,15 @@
     "postinstall:prisma": "prisma generate",
     "postinstall:create-data": "tsx ./build-scripts/create-data.ts",
     "postinstall": "run-p postinstall:*",
-    "lint": "next lint",
+    "lint:next": "next lint",
+    "lint:prisma:validate": "prisma validate",
+    "lint:prisma:format": "prisma format --check",
+    "lint:prisma": "run-p lint:prisma:*",
+    "lint": "run-p lint:*",
+    "types": "tsc --noEmit",
     "start": "next start",
     "test": "vitest",
-    "format": "prettier . --write && prisma format"
+    "format": "prisma format"
```

Comment:
What is the background behind removing prettier here?

I personally use prettier in the editor but having a command sounds good too unless there is a problem with it?

Reply:
`format` in the top-level package runs prettier -- I figured we only needed it in one place (plus the chatbot doesn't define it as `format` either).

If you prefer to have it in here as well, happy to add it back.

Reply:
sounds good